### PR TITLE
[TASKMGR] Application Page: Fix "Go To Process" item selection

### PR DESCRIPTION
**Pull request by TAN-Gaming**
## Purpose

Fix "Go To Process" selecting wrong process in the process page

### CODE REVIEW

File: base/applications/taskmgr/applpage.c [hunk 1]
```diff
@@ -944,6 +944,8 @@ void ApplicationPage_OnGotoProcess(void)
         i = ProcGetIndexByProcessId(dwProcessId);
         if (i != -1)
         {
+            ListView_SetSelectionMark(hProcessPageListCtrl, i);
```
Comment (HBelusca) [inline base/applications/taskmgr/applpage.c:947]: How does this differ with the `SetItemState` from below? (is there also some sort of visual difference, with and without that line?)

Comment (TAN-Gaming) [reply base/applications/taskmgr/applpage.c:947]: Yes.

Comment (binarymaster) [reply base/applications/taskmgr/applpage.c:947]: I cannot reproduce this problem on unpatched Task Manager in Windows 10. Need to check on XP/2003.

**Comment by TAN-Gaming** [comment]
The problem maybe in another module, taskmgr works fine on Windows.

**Pull request closed without merging**